README.rst

```diff
@@ -140,66 +142,63 @@ __ `FIWARE Policy Manager GE: Cloto`_
 
 Configuration file - Cloto
 --------------------------
-Before starting the rule engine, you should edit settings.py located at cloto folder or in ``/etc/sysconfig/fiware-cloto.cfg``.
-Constants you eed to complete are:
+The configuration used by the fiware-cloto component is read from the configuration file.
+This file is located here:
 
-- All in ``# OPENSTACK CONFIGURATION``: Openstack information (If you provide this information in the install script you do not need to edit)
-- RABBITMQ_URL: URL Where RabbitMQ is listening (no port needed, it uses default port)
-- CONTEXT_BROKER_URL: URL where Orion Context Broker is listening
-- NOTIFICATION_URL: URL where notification service is listening (This service must be implemented by the user)
+``/etc/fiware.d/fiware-cloto.cfg``
 
-In addition you could modify other constants like NOTIFICATION_TIME, or DEFAULT_WINDOW_SIZE.
+MYSQL cloto configuration must be filled before starting fiware-facts component, user and password are empty by default.
+You can copy the `default configuration file <fiware_cloto/cloto_settings/fiware_cloto.cfg>`_ to the folder defined for your OS, and
```

Comment:
This link is broken...

`default configuration file <fiware_cloto/cloto_settings/fiware_cloto.cfg>`_